.cspell.json

```diff
@@ -122,6 +122,7 @@
         "Boothole",
         "dbxupdate",
         "FileFlagsMask",
-        "markdownlint"
+        "markdownlint",
+        "xffd"
```

Comment:
Where was this needed?

Reply:
This was used in edk2_setup.py. I am not sure why it start to bubble up in this PR given I have not touched it.